docs/source/releases/index.rst

```diff
@@ -16,14 +16,6 @@
 Release Notes
 *************
 
-Latest (version on cutting edge of git)
----------------------------------------
-
-.. toctree::
-   :maxdepth: 1
-
-   latest
-
```

Comment:
Do not merge this yet.  The *.rst release note check should actually be failing because this is modified and latest.rst was deleted.  Make sure that check is failing before merging.

Reply:
![image](https://github.com/user-attachments/assets/88209eb3-ba04-4d79-b790-c8b42df22af2)

Reply:
The RST modified check successfully failed, and all others passed.  Will override and merge, since we were intentionally chaging the RST files (and we should not have to change them again)

Reply:
![image](https://github.com/user-attachments/assets/f551b181-806f-4dad-82a7-aa2a963a7010)